src/libraries/Microsoft.Extensions.Logging.Console/tests/ConsoleLoggerExtensionsTests.cs

```diff
@@ -2,17 +2,17 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System;
-using System.Collections.Generic;
-using System.Text.Json;
-using System.Text.Encodings.Web;
-using System.IO;
 using Microsoft.Extensions.Logging.Abstractions;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Console;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Options;
+using System;
```

Comment:
Shouldn't the system-namespaces come first, then the rest?
Cf. [C# Coding Style Pt. 6](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md)

Reply:
Yes. It is in our `.editorconfig` as well:

https://github.com/dotnet/runtime/blob/84c2974ab6b4205366b50484b21416ab920d9424/.editorconfig#L82

@maryamariyan - what did you use to sort these? Visual Studio or something else? VS should have respected the .editorconfig setting.

Reply:
Used VS but it put Microsoft.Extensions namespaces above System.
It also was wrong in the sense that "Remove and Sort unused usings" removed more than it should have but that I fixed manually.

I'll fix this up.